app/views/users/index.html.erb

```diff
@@ -1,9 +1,16 @@
 <h1>Welcome to users index page</h1>
 <% @users.each do |user| %>
-  <section>
-    <img src="<%= user.photo %>" alt="User image">
-    <h2><%= user.name %></h2>
-    <h3>Number of posts: <%= user.posts_counter %></h3>  
-  </section>
+  <%= link_to user_path(user.id) do %>
+  <div class="user_card">
+    <section>
+      <img src="<%= user.photo %>" alt="User image">
+    </section>
+    <section class="name_posts">      
+      <h2 class="user_name"><%= user.name %></h2>
+      <h3 class="user_posts">Number of posts: <%= user.posts_counter %></h3>  
+    </section>
+  </div>
+  <% end %>
 <% end %>
```

Comment:
- [Optional] Please always make sure to set a default root route to prevent the default Rails startup page when routes to  `http://localhost:3000` The user's index page could have been set as the default routes as    `root "users#index"`

Reply:
Done.